app/routes/events/components/EventAdministrate/Abacard.js

```diff
@@ -1,62 +1,69 @@
 // @flow
 
-import React from 'react';
-import { reduxForm, Field } from 'redux-form';
-import { SelectInput } from 'app/components/Form';
-import Button from 'app/components/Button';
-import type { ID, EventPool, EventRegistrationPresence } from 'app/models';
-import type { ReduxFormProps } from 'app/types';
-
-type Props = {
-  eventId: ID,
-  updatePresence: (number, number, string) => Promise<*>,
-  pools: Array<EventPool>
-} & ReduxFormProps;
-
-const Abacard = ({
-  eventId,
-  handleSubmit,
-  updatePresence,
-  pools,
-  invalid,
-  pristine,
-  submitting
-}: Props) => {
-  const onSubmit = (
-    registrationId: ID,
-    presence: EventRegistrationPresence
-  ) => {
-    updatePresence(eventId, registrationId, presence);
-  };
-  return (
-    <div>
-      <form onSubmit={handleSubmit(onSubmit)}>
-        <Field
-          name="user"
-          component={SelectInput.AutocompleteField}
-          filter={['users.user']}
-          placeholder="Bruker"
-          label="Bruker"
-        />
-        <Button type="submit" disabled={invalid || pristine || submitting}>
-          Registrer
-        </Button>
-      </form>
-    </div>
-  );
+import * as React from 'react';
+import cx from 'classnames';
+import SearchPage from 'app/components/Search/SearchPage';
+import type { SearchResult } from 'app/reducers/search';
+import styles from './Abacard.css';
+
+// $FlowFixMe
+import goodSound from '../../../../assets/good-sound.mp3';
+
+type State = {
+  showCompleted: boolean
 };
 
-function validateForm(data) {
-  const errors = {};
+class Abacard extends React.Component<*, State> {
+  state = {
+    showCompleted: false
+  };
 
-  if (!data.user) {
-    errors.user = 'Bruker er påkrevet';
-  }
+  showCompleted = () => {
+    this.setState({ showCompleted: true });
+    setTimeout(() => this.setState({ showCompleted: false }), 2000);
+  };
+
+  handleSelect = (result: SearchResult) => {
+    this.props.handleSelect(result).then(
+      () => {
+        const sound = new window.Audio(goodSound);
+        sound.play();
+        this.showCompleted();
+        return this.props.clearSearch();
+      },
+      err => {
+        const payload = err.payload.response.jsonData;
+        if (payload.error_code === 'not_registered') {
```

Comment:
These are "camelCased" by the backend before they are sent. So you should use `payload.errorCode`.